meeting_scheduler/meetingscheduler/src/main/java/com/spring/rest/meetingscheduler/entity/MeetingDetail.java

```diff
@@ -0,0 +1,105 @@
+package com.spring.rest.meetingscheduler.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+import java.sql.Date;
+import java.sql.Time;
+
+@Entity
+@Table(name = "meeting_detail")
+public class MeetingDetail {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "meeting_detail_id")
+    private int meetingDetailId;
+
+    @OneToOne
+    @JoinColumn(name = "meeting_id")
+    private Meeting meeting;
+
+    @Column(name = "meeting_date")
+    private Date meetingDate;
+    @Column(name = "meeting_start_time")
+    private Time meetingStartTime;
+
+    @Column(name = "meeting_end_time")
+    private Time meetingEndTime;
+
+    @ManyToOne(cascade = {
```

Comment:
give pointers why defaults are not used and whats the use of below mentioned cascades

Reply:
In default cascades all the cascades will be attached i.e., if we delete a row the corresponding value will be deleted in other table too. So in our case this meeting detail entity has employee ID. There is a specific use case that we should be able to remove people from the meeting. So in case we have default cascades if we remove people from it will also delete the employee. So to avoid that I have removed CascadeType.Remove and added others.